File: ScantronPDFProducer/libs/PDFjet.78/Example_19.java
import java.lang.*;
import java.text.*;
import java.io.*;
import java.util.*;

import com.pdfjet.*;


/**
 *  Example_19.java
 *
 */
class Example_19 {

    public Example_19() throws Exception {

        try {

            FileOutputStream fos = new FileOutputStream("Example_19.pdf");
            BufferedOutputStream bos = new BufferedOutputStream(fos);

            PDF pdf = new PDF( bos );

            FileInputStream fis = new FileInputStream( "Example_03.pdf" );

/*
            FileInputStream fis =
                    new FileInputStream( "PDF32000_2008.pdf" );
*/
            BufferedInputStream bis = new BufferedInputStream( fis );
            List< PDFobj > objects = pdf.read(bis);
            for ( PDFobj obj : objects ) {
                for ( int i = 0; i < obj.dict.size(); i++ ) {
                    System.out.println(obj.dict.get(i));
                }
                System.out.println();
            }
            bis.close();

            pdf.flush();
            bos.close();
        }
        catch ( Exception e ) {
            e.printStackTrace();
        }

    }


    public static void main(String[] args) {
        try {
            new Example_19();
        } catch (Exception e) {
            e.printStackTrace();
        }
    }

}   // End of Example_19.java

File: ScantronPDFProducer/libs/PDFjet.78/Example_19.cs

using System;
using System.Collections.Generic;
using System.IO;

using PDFjet.NET;


/**
 *  Example_19.cs
 *
 */
class Example_19 {

    public Example_19() {

        try {

            FileStream fos = new FileStream("Example_19.pdf", FileMode.Create);
            BufferedStream bos = new BufferedStream(fos);

            PDF pdf = new PDF( bos );

            FileStream fis = new FileStream( "Example_03.pdf", FileMode.Open );

            // FileInputStream fis = new FileInputStream( "PDF32000_2008.pdf" );

            BufferedStream bis = new BufferedStream( fis );
            List< PDFobj > objects = pdf.read(bis);
            for ( int j = 0; j < objects.Count; j++ ) {
                PDFobj obj = ( PDFobj ) objects[j];
                for ( int i = 0; i < obj.dict.Count; i++ ) {
                    Console.WriteLine(obj.dict[i]);
                }
                Console.WriteLine();
            }
            bis.Close();

            pdf.Flush();
            bos.Close();
        }
        catch ( Exception e ) {
            Console.WriteLine(e.StackTrace);
        }

    }


    public static void Main(String[] args) {
        try {
            new Example_19();
        } catch (Exception e) {
            Console.WriteLine(e.StackTrace);
        }
    }

}   // End of Example_19.cs
